_topics/en/compact-block-filters.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Compact block filters
+
+## Optional.  An entry will be added to the topics index for each alias
+aliases:
+  - BIP157
+  - BIP158
+  - Neutrino protocol
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Privacy Enhancements
+  - Lightweight Client Support
+  - P2P Network Protocol
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+excerpt: >
+  **Compact block filters** are a condensed representation of the
+  contents of a block that allow wallets to determine whether the block
+  contains any transactions affecting the wallet.
+
+## Optional.  Use Markdown formatting.  Multiple paragraphs.  Links allowed.
+extended_summary: |
+  A full node with a copy of a block and the data necessary to validate
```

Comment:
I think we can just say 'a full node' here and assume that it has the block and the data necessary to validate the block.

Reply:
I'm not completely opposed to that, but I think it's helpful to mention those conditions to help explain why pruned nodes can't create filters for blocks where they've already pruned the block and undo data.

Reply:
That feels a bit too in the weeds for me, but happy to go either way.

Reply:
I think this is less clear than the original. "some related data" just begs the question "which data?" I've reverted to your original version.